### PR TITLE
Refactor client to reduce sonar bugs

### PR DESCRIPTION
**Pull request by jt2594838**

### CODE REVIEW

**Comment by jt2594838** [comment]
Replacing the `System.out` with a more delicate printer is more graceful, though I wonder how to do this. Advices are welcomed.

File: iotdb-cli/src/main/java/org/apache/iotdb/cli/client/AbstractClient.java [hunk 1]
```diff
@@ -144,6 +145,12 @@
 
   protected static ServerProperties properties = null;
 
+  private static boolean printHeader = false;
+  private static int displayCnt = 0;
+
+  private static final PrintStream SCREEN_PRINTER = new PrintStream(System.out);
+  private static boolean showException = false;
```
Comment (MyXOF) [inline iotdb-cli/src/main/java/org/apache/iotdb/cli/client/AbstractClient.java]: I did not find any statement to set showException =true, so showException is always false?

Comment (jt2594838) [reply iotdb-cli/src/main/java/org/apache/iotdb/cli/client/AbstractClient.java]: Yes, as some may think printing stack-trace in console is somehow unfriendly. Maybe we can add this to options in the future if needed.

Comment (MyXOF) [reply iotdb-cli/src/main/java/org/apache/iotdb/cli/client/AbstractClient.java]: So you can add some comments to indicate `showException` is not used currently and we will add this to options in the future if needed.

Comment (jt2594838) [reply iotdb-cli/src/main/java/org/apache/iotdb/cli/client/AbstractClient.java]: Added.

**Pull request merged**